tracer/test/test-applications/Directory.Build.props

```diff
@@ -20,6 +20,8 @@
     <DefineConstants Condition="'$(TestAllPackageVersions)'=='true' AND '$(IncludeMinorPackageVersions)'=='true'">$(DefineConstants);TEST_ALL_MINOR_PACKAGE_VERSIONS</DefineConstants>
     <!-- Hide warnings for EOL .NET Core targets (e.g. netcoreapp3.0) -->
     <CheckEolTargetFramework>false</CheckEolTargetFramework>
+    
+    <NoWarn>SYSLIB0014</NoWarn>
```

Comment:
```suggestion
    <!-- WebClient is obsolete -->
    <NoWarn>SYSLIB0014</NoWarn>
```

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1885/commits/242c3c813e05991fdf1543aa6f7574ce577c6805